lib/models/purchase_params.dart

```diff
@@ -0,0 +1,38 @@
+import 'google_product_change.dart';
+import 'package_wrapper.dart';
+import 'promotional_offer.dart';
+import 'purchasable.dart';
+import 'store_product_wrapper.dart';
+import 'subscription_option_wrapper.dart';
+
+/// Parameters for making a purchase.
+class PurchaseParams {
+  final Purchasable purchasableItem;
+  final GoogleProductChangeInfo? googleProductChangeInfo;
+  final bool? googleIsPersonalizedPrice;
+  final PromotionalOffer? promotionalOffer;
```

Comment:
Right now this allows potentially passing both a `googleProductChangeInfo` and a `promotionalOffer` at the same time, which doesn't make much sense... But I think it's ok? The platform where the other parameter doesn't work will be ignored.